tokio/src/runtime/task/core.rs

```diff
@@ -60,7 +60,7 @@ pub(crate) struct Header {
     /// Task state.
     pub(super) state: State,
 
-    pub(super) owned: UnsafeCell<linked_list::Pointers<Header>>,
+    pub(super) owned: linked_list::Pointers<Header>,
```

Comment:
This is ok since `Pointers` already contains an `UnsafeCell`.

https://github.com/tokio-rs/tokio/blob/922fc91408d68f0a6e66e14c99cb47a16e8653af/tokio/src/util/linked_list.rs#L70-L73